JavaScript and Ajax/StudentDeleteRemote.page

```diff
@@ -0,0 +1,45 @@
+<apex:page controller="StudentDeleteRemote">
+    <script type="text/javascript">
+    function deleteStudents() {
+        try{
+            const queryString = window.location.search;
+            const urlParams = new URLSearchParams(queryString);
+            var records = urlParams.getAll('id');
+            Visualforce.remoting.Manager.invokeAction(
+                '{!$RemoteAction.StudentDeleteRemote.deleteStudent}',
+                records, 
+                function(result, event){
+                    if (event.status) {
```

Comment:
what if the status code is 500 ??

Reply:
500 Internal Server Error server error response code indicates that the server encountered an unexpected condition that prevented it from fulfilling the request.